src/lib/gw2/itemStats.js

```diff
@@ -0,0 +1,90 @@
+import { queryItemAttributes } from 'gw2itemstats';
+
+function calcAttribute (base, modifier) {
+  return base + modifier;
```

Comment:
Yes, this is what gw2wiki are doing,
There should be a way to calc the values off the selectedStat object (which is gw2 api object) but there is no formula posted anywhere and couldnt find it easily